app/controllers/events/view/speakers/create.js

```diff
@@ -11,7 +11,7 @@ export default class extends Controller {
       }
       const newSpeaker = this.model.speaker;
       if (newSpeaker.isEmailOverridden) {
-        newSpeaker.set('email', this.authManager.currentUser.email);
+        newSpeaker.set('email', 'Not Provided');
```

Comment:
No, that is wrong data

Reply:
what should be put here?

Reply:
null

Reply:
When I am setting it to null it throws err -` the server responded with a status of 422 ()`

Reply:
Server needs to allow null emails in case organizers or admin create the speaker

Reply:
> Server needs to allow null emails in case organizers or admin create the speaker

how to do this :worried:

Reply:
Need to change speaker schema to allow null and then add a check that email is not null if a not organizer/admin user is submitting/editing the email

Reply:
after allowing null from server and adding null here, it is not being null, It sets to `user email` on writing `newSpeaker.set('email', null); .` But on writing` newSpeaker.set('email', ' any text'); `the email is set to that text.

Reply:
Adding null it is setting it to owner email , but if it's not null then setting to that text
![scrnli_10_23_2020_8-44-05 PM](https://user-images.githubusercontent.com/56407566/97021501-89f53100-1570-11eb-8bf3-447c3c6a25bb.png)

Reply:
this is happening because of this on server side : 
```
elif (
            data.get('is_email_overridden')
            and has_access('is_organizer', event_id=data['event'])
            and not data.get('email')
        ):
            data['email'] = current_user.email
```

Reply:
What to do here?

Reply:
Will check why this code was added.